packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -83,17 +84,18 @@ export class AngularCompilerPlugin implements Tapable {
   private _angularCompilerOptions: any;
   private _tsFilenames: string[];
   // Should be Program from @angular/compiler-cli instead of any.
-  private _program: any;
+  private _program: ts.Program & any;
```

Comment:
I'm not sure `& any` is even something... Wouldn't that be `any`? Any reason you cannot put `ts.Program` there?

Reply:
When the interface is available in a stable manner, we should be doing `ts.Program & Program`. Meanwhile I wanted to leave a marker there that indicated it's not just `ts.Program`. 

For some of these we should be able to not have the `& any` but for others that won't be possible since methods/props are being used that don't exist on ts.Program.